frontend-react/src/components/ManagePublicKey/ManagePublicKey.tsx

```diff
@@ -1,18 +1,136 @@
-import React from "react";
-import { GridContainer } from "@trussworks/react-uswds";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { GridContainer, Icon, SiteAlert } from "@trussworks/react-uswds";
 
 import { AuthElement } from "../AuthElement";
 import { withCatchAndSuspense } from "../RSErrorBoundary";
+import { USLink } from "../USLink";
+import { showError } from "../AlertNotifications";
 import { MemberType } from "../../hooks/UseOktaMemberships";
+import { validateFileType, validateFileSize } from "../../utils/FileUtils";
+
+import ManagePublicKeyChooseSender from "./ManagePublicKeyChooseSender";
+import ManagePublicKeyUpload from "./ManagePublicKeyUpload";
+
+export const CONTENT_TYPE = "application/x-x509-ca-cert";
+export const FORMAT = "PEM";
+
+function ManagePublicKeySwitchDisplay() {
+    const navigate = useNavigate();
+    const [sender, setSender] = useState("");
+    const [fileContent, setFileContent] = useState("");
+    const [file, setFile] = useState<File | null>(null);
+    const [fileSubmitted, setFileSubmitted] = useState(false);
+
+    // TODO: mocked for now - make the call you need when sending the file
+    const { sendFile } = {
+        sendFile: (data: {
+            contentType?: string;
+            fileContent?: string;
+            file?: File | null;
+        }) => {
+            data = { file: null };
+            return data;
+        },
+    };
+
+    const onSenderSelect = (selectedSender: string) => {
+        setSender(selectedSender);
+    };
+
+    const onBack = () => {
+        navigate(-1);
```

Comment:
Since this is all on one page, I think the `navigate(-1)` would be incorrect since we're not pushing to history state so each `onBack` should really be unsetting the state so we end up at the proper step instead.  Sorry, this is my fault for suggesting `navigate` usage here.

Reply:
So should we redirect to the Resources page? Otherwise back may just display `ManagePublicKeyUpload` if the org only has one sender.

Reply:
We can break this out into a subsequent ticket/pull request of course!  We don't have to hold up this pull request to get everything perfect since the flow is intended to be incomplete at this stage anyway.  BUT this should be fixed prior to when we go live.

Reply:
Will remove and create a new ticket for future sprint.